.github/workflows/master-workflow.yaml

```diff
@@ -1,4 +1,4 @@
-name: Main workflow
+name: Master workflow
 on:
   workflow_dispatch:
```

Comment:
Pour le `workflow_dispatch` je me demande à quoi il servait et si il y avait une utilité. @apomalyn le sais-tu? soit on devrait l'ajouter dans le dev workflow, soit le retirer ici ou si il y a une raison valable qu'il soit sur master mais pas sur dev?

Reply:
Selon [la documentation](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#workflow_dispatch), ça permet de trigger le CI manuellement avec l'API Github, le CLI, ou le site web. Je pense que c'est utile surtout pour retrigger un build qui a échoué.

Reply:
Sorry I didn't saw the notification :sweat_smile: De memoir c'etait pour le trigger a souhait uniquement mais ca n'a jamais ete utilise (toujours de memoire). @XavierPaquet-Rapold tu n'as pas besoin d'un `workflow_dispatch` pour relancer un build echoue :smile:

Reply:
D'acc! Ouais je pense effectivement que ça n'a jamais été utilisé. Mais au pire @XavierPaquet-Rapold mets-le et si jamais ça se voit être utile on sera prêt! Merci @apomalyn